lib/std/array_list.zig

```diff
@@ -149,17 +173,22 @@ pub fn AlignedArrayList(comptime T: type, comptime alignment: ?u29) type {
             return self.swapRemove(i);
         }
 
+        /// Append the slice of items to the list. Copies the contents
+        /// of `items`.
```

Comment:
I was trying to make it clear that it's not a move, and that if `items` contains any pointers they're aliased now. Of course, that's true for all the insertion functions, so it probably either belongs in all of them or none of them. Maybe when the top-level doc has examples, it can be made clear there.